.github/workflows/compilation.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+    types: [run_build]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: ghcr.io/fjtrujy/pspsdk:latest
```

Comment:
This will still require a change before merging

Reply:
yeah!